### PR TITLE
From PRINTF   to LM_F macro

### PR DESCRIPTION
**Pull request by raconte**
It has been deleted the PRINTF macro present in the source code of /fiware_orion/src/lib/ngsi/ngsi9/ngsi10 and replaced using LM_F

### CODE REVIEW

File: src/lib/ngsi/AttributeAssociation.cpp [hunk 1]
```diff
@@ -25,6 +25,8 @@
 #include <stdio.h>
```
Comment (fgalan) [inline src/lib/ngsi/AttributeAssociation.cpp]: Not all the PRINTF has been removed:

```
$ grep PRINTF -R src/* | wc -l
43
```

File: src/lib/ngsi/AttributeAssociation.cpp [hunk 2]
```diff
@@ -25,6 +25,8 @@
...
 #include <string>
```
Comment (fgalan) [inline src/lib/ngsi/AttributeAssociation.cpp]: Once the work is finished, the following could be removed from globals.h:

```
/* ****************************************************************************
*
* PRINTF - 
*/
#define PRINTF printf
```

**Comment by raconte** [comment]
Fixed in 71e14ef d7c7322 d698f58 dadb320 2686427 eb66d02 c1aac8c a00dc70 20dd35d 075f5e 896163b dd543d6 ab1c881 ab23304 7e35954 3429bcf 8e7e7ff dd9cfd9 f2ba981 5d873f6 cc0cf44  9637297 e5c4a48 26543ee 4a3fa1e bcfbd78 bd92135 161a263 76e794b 0f9f6e1

**Comment by fgalan** [comment]
LGTM

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**